docs/approach.rst

```diff
@@ -84,12 +82,7 @@ transforms the data by default.
 The BOLD data are transformed as :math:`log(|S|+1)`, where :math:`S` is the BOLD signal.
 The echo times are also multiplied by -1.
 
-`Decay model fit code`_
-
-.. _Decay model fit code: https://tedana.readthedocs.io/en/latest/generated/tedana.decay.fit_decay.html#tedana.decay.fit_decay
-
-
-.. note::
+.. tip::
```

Comment:
@emdupre, do you know the right CSS code to inject into our theme_override.css file to control the colors of our admonition boxes? I spent a fair amount of time trying to get `note` and `see also` to have different colors, but couldn't find the right entity to edit.

Reply:
The CSS incantation is:

```
div.admonition.note {
  background: #hex-color
}
.p.admonition.title {
  background: #darker-hex-color
}
```

Updating `note` to `warning` or any other kind of admonition. We need to add in a new CSS file in `static` to hold these, like we do in Nilearn. In a few days I can make a PR on your PR if that's helpful :smile_cat:

Reply:
Actually, I think the `theme_override.css` _should_ work. Let's try it there first ?!

Reply:
Thanks!

Reply:
The box background works, but not the title.

Reply:
Try:

```
.rst-content .note .admonition-title {
  background: #darker-hex-color
}
```

?

Reply:
Also, the current color is just for testing, right ? :sweat_smile:

Reply:
It was, and somehow during my merge commit I pushed a change that I'd been specifically _not_ committing, but 🤷‍♂️.

Why, you don't like it? I think it's fancy!

![image](https://user-images.githubusercontent.com/8228902/87366939-e72c2000-c547-11ea-9a9e-5e24c2960f59.png)

So fancy.

Reply:
Your recommendation worked!

![image](https://user-images.githubusercontent.com/8228902/87367174-47bb5d00-c548-11ea-9099-e14dc8c3297d.png)

Reply:
Great !! Now let's iterate on the colors a bit :joy: :man_artist:

Reply:
I'm going to drop the Notes change but add one for Caution boxes to make it look like below:

![image](https://user-images.githubusercontent.com/8228902/87369361-51e05a00-c54e-11ea-9900-b3109c290b21.png)

It's only used once in our documentation, but I think it'll be a good example as we circle back to this. The issue being that several different boxes are blue (like Notes) and sometimes those boxes will come up near each other and when they're the same color it doesn't look great. Also, if we use Napoleon to use admonitions for some of our docstring sections, at least Notes and References are the same color and right next to each other.

WDYT?